tenable/sc/scan_instances.py

```diff
@@ -258,6 +264,12 @@ def list(self, fields=None):
             params['fields'] = ','.join([self._check('field', f, str) 
                 for f in fields])
 
+        if start_time:
+            params['startTime'] = str(start_time)
```

Comment:
str(start_time) should be self._check('start_time', start_time, int)

Reply:
Good call.  Fixed.